### DISCUSSION TITLE
General questions

### DISCUSSION THREAD

**Discussion by JBehnke98** [tryzealot/zealot#1525] [Q&A] [answered]
Hello everyone, 

My team and I want to use Zealot. There are some functions that are important to us. Can anyone help me and tell me whether Zealot supports these functions? 

1. invitation to an app version via mail notification
2. access restrictions with a password
3. projects/apps should only be visible to user groups that have been invited to them
4. creation of several tester groups (internal & external) 
5. a user should be able to select several different test versions 
6. at least 3 previous release versions should still be available for control purposes if possible
7. a connection to Testflight should be possible
8. management of iOS devices within Zealot
9. it should be possible to manage the respective release versions. If necessary, withdraw a version and activate the previous version.
10. provision of a test version that supports different backends
11. It should be possible to activate a Crashlytics framework, which then e.g. tracks requests to the backend and enables evaluations 
12. Is there an easy way to migrate existing app versions and test tracks from other tools or is this only possible manually?

Many thanks for your help!

**Comment by icyleaf** [comment]
Thank you for your attention to Zealot. Here are the answers to your questions.

> invitation to an app version via mail notification

No, The email is currently only used for email invitation confirmation, and the new version notification service can use web hooks.

> access restrictions with a password

Yes, For each App's channel could set password.

> projects/apps should only be visible to user groups that have been invited to them

Yes, For each App, there are [collaborator permission settings](https://zealot.ews.im/docs/next/user-guide/administrator/permissions#collaborators).

> a user should be able to select several different test versions

Refer to the previous answer.

> at least 3 previous release versions should still be available for control purposes if possible

Yes, App owner can set the maximum number of reservations per channel, with no limit by default.

> a connection to Testflight should be possible

No, This may be possible but still in the conception stage.

> management of iOS devices within Zealot

Yes, See [Apple Team](https://zealot.ews.im/docs/next/user-guide/administrator/apple-team).

> it should be possible to manage the respective release versions. If necessary, withdraw a version and activate the previous version.

No, No withdraw, Delete it instead.

> provision of a test version that supports different backends

Sorry, i don't understand what you mean.

> It should be possible to activate a Crashlytics framework, which then e.g. tracks requests to the backend and enables evaluations

No, Is this feature so important for a distribution service?

> Is there an easy way to migrate existing app versions and test tracks from other tools or is this only possible manually?

No, we offer APIs to facilitate developers in customizing. I only known Microsoft AppCenter, what do your team use for?

**Reply by JBehnke98** [reply]
Many thanks for your answers! :)

Based on your answers, new questions have come up on my side:

1. “No, the email is currently only used for email invitation confirmation, and the new version notification service can use web hooks.” – How does the invitation to the app work then? Do you have to provide the login data to the person you invited? 
2. Is it possible to create several tester groups (internal & external) with different permissions? 
3. “Yes, the app owner can set the maximum number of reservations per channel, with no limit by default.” – Do you mean by reservations the release versions? 
4. “It should be possible to manage the respective release versions. If necessary, withdraw a version and activate the previous version.” – “No, no withdraw; delete it instead.” – Okay, but if you delete the current version, is it possible to activate the previous one?
5. “Provision of a test version that supports different backends” – “Sorry, I don't understand what you mean.” – We work with test versions that are called “XYZ - Production”, “XYZ - Test”, “XYZ - Stage”, for example, and there are different backends behind them. Is this also possible via Zealot?

**Reply by icyleaf** [reply, marked as answer]
> How does the invitation to the app work then? Do you have to provide the login data to the person you invited?

Zealot's current service positioning is aimed at businesses or organizations. If the setting for open registration is turned off, then email invitations can be used for user management within the application.

> Is it possible to create several tester groups (internal & external) with different permissions?

No.

> Do you mean by reservations the release versions?

Any type of version.

> “It should be possible to manage the respective release versions. If necessary, withdraw a version and activate the previous version.” – “No, no withdraw; delete it instead.” – Okay, but if you delete the current version, is it possible to activate the previous one?

Every version has a unique link to visit.

> “Provision of a test version that supports different backends” – “Sorry, I don't understand what you mean.” – We work with test versions that are called “XYZ - Production”, “XYZ - Test”, “XYZ - Stage”, for example, and there are different backends behind them. Is this also possible via Zealot?

Yes, The application itself is based on the free combination of scheme and channel dimensions, allowing you to create any you wish.

Base all your question, You can get the answer after read the document and try on the demo service please.
